### ISSUE TITLE
1.5.1 change log

### ISSUE THREAD

**Issue by niemyjski** [mgonto/restangular#1116]
Could someone please update the releases changelog and update the other change log and provide a more user friendly change log with a high level overview of what all changed and any breaking changes that occurred.

**Comment by grabbou** [comment]
Yes. We are in a process of writing everything down. @mgonto check PM

**Comment by niemyjski** [comment]
Thank you

On 4/8/15, Mike Grabowski <notifications@github.com> wrote:
> Yes. We are in a process of writing everything down. @mgonto check PM
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/mgonto/restangular/issues/1116#issuecomment-91009448


-- 
Thanks
-Blake Niemyjski
